plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java

```diff
@@ -184,6 +184,14 @@
     public static final String ICEBERG_TABLE_TYPE_NAME = "table_type";
     public static final String ICEBERG_TABLE_TYPE_VALUE = "iceberg";
 
+    public static final String HUDI_TABLE_TYPE_NAME = "type";
+    private static final String HUDI_TABLE_TYPE_COPY_ON_WRITE = "cow";
+    private static final String HUDI_TABLE_TYPE_MERGE_ON_READ = "mor";
+    private static final String HUDI_PARQUET_INPUT_FORMAT = "org.apache.hudi.hadoop.HoodieParquetInputFormat";
+    private static final String HUDI_PARQUET_REALTIME_INPUT_FORMAT = "org.apache.hudi.hadoop.realtime.HoodieParquetRealtimeInputFormat";
```

Comment:
you can use regular class objects here

Reply:
please see comment:  https://github.com/trinodb/trino/pull/15551#discussion_r1064867413